tests_end_to_end/test_e2e_demisto_sdk_flow_playbook.py

```diff
@@ -0,0 +1,113 @@
+import random
+from os import path
+
+import e2e_tests_utils
+from demisto_client.demisto_api.rest import ApiException
+
+from demisto_sdk.commands.download.downloader import Downloader
+from demisto_sdk.commands.format.format_module import format_manager
+from demisto_sdk.commands.generate_docs import generate_playbook_doc
+from demisto_sdk.commands.upload.uploader import Uploader
+from demisto_sdk.commands.validate.validate_manager import ValidateManager
+
+
+def test_e2e_demisto_sdk_flow_playbook_testsuite(tmpdir):
+    # Importing TestSuite classes from Demisto-SDK, as they are excluded when pip installing the SDK.
+    e2e_tests_utils.cli(f'mkdir {tmpdir}/git')
+    e2e_tests_utils.git_clone_demisto_sdk(destination_folder=f'{tmpdir}/git/demisto-sdk', sdk_git_branch='testsuite-playbook')
+    from TestSuite.playbook import Playbook
+    from TestSuite.repo import Repo
+
+    repo = Repo(tmpdir)
+
+    unique_id = random.randint(1, 1000)
+    pack_name = 'foo_' + str(unique_id)
```

Comment:
create_pack adds a serial number automatically afaik

Reply:
I'm (re)using the same folder. Without this - I couldn't do successive runs.